packages/api/__tests__/specs/article.ts

```diff
@@ -171,10 +166,14 @@ describe('Articles', () => {
           id: articleIds[0]
         }
       })
+
       expect(updatedArticle).toMatchSnapshot({
         data: {
           updateArticle: {
-            id: expect.any(String)
+            id: expect.any(String),
+            latest: expect.objectContaining({
+              slug: expect.any(String)
```

Comment:
You're simplifying it here because there's no need to test for it matching exact case?

Reply:
See https://github.com/wepublish/wepublish/pull/644#discussion_r929926385